src/access/wac.ts

```diff
@@ -408,3 +409,40 @@ export async function setAgentResourceAccess<T extends WithServerResourceInfo>(
     options
   );
 }
+
+/**
+ * Set the Access modes for a given Group to a given Resource.
+ *
+ * Important note: if the target resource did not have a Resource ACL, and its
+ * Access was regulated by its Fallback ACL, said Fallback ACL is copied to create
+ * a new Resource ACL. This has the side effect that the next time the Fallback
+ * ACL is updated, the changes _will not impact_ the target resource.
+ *
+ * If the target Resource's Access mode cannot be determined, e.g. the user does
+ * not have Read and Write access to the target Resource's ACL, or to its
+ * fallback ACL if it does not have a Resource ACL, then `null` is returned.
+ *
+ * @param resource The Resource for which Access is being set
+ * @param agent The Group for whom Access is being set
+ * @param access The Access being set
+ * @param options Optional parameter `options.fetch`: An alternative `fetch` function to make the HTTP request, compatible with the browser-native [fetch API](https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/fetch#parameters).
+ * @returns The Resource, with its ACL updated, or null if the new Access could not
+ * be set.
+ */
+export async function setGroupResourceAccess<T extends WithServerResourceInfo>(
```

Comment:
As in the other review I'd call this

```suggestion
export async function setGroupAccess<T extends WithServerResourceInfo>(
```

Reply:
Same as for the previous PR #807, I think this should remain `setGroupResourceAccess`, and we can see how to design the default access in the universal API